src/main/java/capstone/examlab/questions/service/QuestionsServiceImpl.java

```diff
@@ -71,30 +71,35 @@ public boolean addQuestionsByExamId(Long examId, QuestionUploadInfo questionUplo
                 .commentary(questionUploadInfo.getCommentary())
                 .commentaryImagesIn(questionUploadInfo.getCommentaryImagesTextIn())
                 .commentaryImagesOut(questionUploadInfo.getCommentaryImagesTextOut())
-                .tagsMap(questionUploadInfo.getTagsMap())
+                .tagsMap(questionUploadInfo.getTags())
                 .build();
         questionsRepository.save(question);
```

Comment:
```suggestion
        questionsRepository.save(question).getId();
```
해당 메서드가 바로 저장된 객체를 반환합니다!
저장하고 다시 찾아서 반환하기보다 이렇게 한번에 Id를 반환하록 하는건 어떨까요?
```
  org. springframework. data. repository. CrudRepository<T, ID>
public abstract <S extends T> S save(     S entity )
```

Reply:
얘기하신부분이 한번에 요구하는 과정이 다 이루어져서 좋은것 같네요
수정해놓을게요